templates/docs/dot-env.md

```diff
@@ -0,0 +1,19 @@
+# .env Support
+
```

Comment:
To have consistency with the other pages, you should add `<%= sinceVersion("0.10.3") %>` helper here.